collectiondb.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+if (isset($_POST['name'])) {
```

Comment:
This is putting too much trust in the hands of the users, although the fields are required, they could easily remove that using the inspector. Suggest making sure the other fields are actually populated before inserting them

Reply:
Adjusted if to account for required fields